### PR TITLE
Prefer Girder to Midas in README

### PR DESCRIPTION
**Pull request by mgrauer**
ping @thewtex @jcfr

### CODE REVIEW

**Comment by mgrauer** [comment]
@jamiesnape PTAL

File: README.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ _Server component of the Midas Platform_
 [![PHP Dependencies](https://www.versioneye.com/user/projects/54fdf4ad4a10649b1b000002/badge.svg)](https://www.versioneye.com/user/projects/54fdf4ad4a10649b1b000002)
 [![StyleCI](https://styleci.io/repos/18849230/shield)](https://styleci.io/repos/18849230)
 
+## Use Girder Instead of Midas ##
+
+Prefer [Girder](https://github.com/girder/girder), which is a more modern and better supported platform than Midas.  There is [documentation](https://github.com/girder/girder/tree/master/scripts/midas) for migrating a Midas3 instance to Girder.  Help is available, documented in the [Girder GitHub repo](https://github.com/girder/girder).
```
Comment (jamiesnape) [inline README.md]: Midas -> Midas Platform
Midas3 -> Midas Server

**Comment by jamiesnape** [comment]
While you are editing the README, remove the coveralls, codeclimate, and versioneye badges.

File: README.md (hunk 1 above)
Comment (jcfr) [inline README.md]: documented in the [Girder Readthedocs](http://girder.readthedocs.org).

Comment (jamiesnape) [inline README.md]: I do not see any migration documentation there.

**Comment by cpatrick** [comment]
image

**Comment by mgrauer** [comment]
ping @jamiesnape

File: README.md [hunk 2]
```diff
@@ -5,14 +5,17 @@
...
-[![Code Climate](https://img.shields.io/codeclimate/github/midasplatform/Midas.svg)](https://codeclimate.com/github/midasplatform/Midas)
 [![Packagist Version](https://img.shields.io/packagist/v/midas-platform/midas-server.svg)](https://packagist.org/packages/midas-platform/midas-server)
 [![Packagist License](https://img.shields.io/packagist/l/midas-platform/midas-server.svg)](https://packagist.org/packages/midas-platform/midas-server)
-[![PHP Dependencies](https://www.versioneye.com/user/projects/54fdf4ad4a10649b1b000002/badge.svg)](https://www.versioneye.com/user/projects/54fdf4ad4a10649b1b000002)
 [![StyleCI](https://styleci.io/repos/18849230/shield)](https://styleci.io/repos/18849230)
 
+## Use Girder Instead of Midas ##
+
+Prefer [Girder](https://github.com/girder/girder), which is a more modern and better supported platform than Midas Platform.  Help is available for Girder, documented in the [Girder GitHub repo README](https://github.com/girder/girder/blob/master/README.rst), and Girder's [readthedocs](http://girder.readthedocs.io/en/latest/) page.
```
Comment (jamiesnape) [inline README.md]: Officially it is "Read the Docs". Change the "Midas" in the section heading to "Midas Platform."

**Comment by mgrauer** [comment]
Thanks for the corrections, they are done.

**Comment by jamiesnape** [comment]
LGTM.

**Comment by jcfr** [comment]
:+1:

**Pull request merged**